contracts/bridge/src/truth/eth/BeaconLightClient.sol

```diff
@@ -185,8 +194,8 @@ contract BeaconLightClient is BeaconChain, Bitfield, StorageVerifier {
                "!execution_payload_state_root"
         );
 
-        uint64 current_period = compute_sync_committee_period(update.attested_header.slot);
-        bytes32 current_sync_committee_root = sync_committee_roots[current_period];
+        uint64 signature_period = compute_sync_committee_period(update.signature_slot);
+        bytes32 current_sync_committee_root = sync_committee_roots[signature_period];
 
         require(current_sync_committee_root != bytes32(0), "!missing");
```

Comment:
This is different with spec:

https://github.com/ethereum/consensus-specs/pull/2805/files#diff-5e27a813772fdd4ded9b04dec7d7467747c469552cd422d57c1c91ea69453b7dR206-R209

```
    assert signature_period in (finalized_period, finalized_period + 1)
...
    if signature_period == finalized_period:
        sync_committee = store.current_sync_committee
    else:
        sync_committee = store.next_sync_committee
```

Any historical sync_committee_root is able to sign? This might be too loose to fit in spec. Better to compare with `current_period `, if equal use `current_period`, otherwise compare equal and use `current_period + 1`

Reply:
Fixed